linetools/lists/parse.py

```diff
@@ -267,25 +274,38 @@ def parse_morton03(orig=False, tab_fil=None):
                 #xdb.set_trace()
                 # Line index
                 elmi.append(kk)
-            # ION
+
+            # ISOTOPE and ION
             try:
```

Comment:
Can you leave a comment here that the try-accept is necessary to get around the bad byte in Morton00?